UITable/UITable/Controller/DetailController.swift

```diff
@@ -66,37 +59,70 @@ class DetailController: UIViewController {
     
     @IBAction func openWikiButtonTapped(_ sender: UIButton) {
         let alert = UIAlertController(title: "Choose browser", message: nil, preferredStyle: .actionSheet)
-        alert.addAction(getAction(title: "UIWebView", idController: "uiWebID"))
-        alert.addAction(getAction(title: "WKWebView", idController: "wkWebID"))
-        alert.addAction(getAction(title: "SFSafariView", idController: "sfWebID"))
+        alert.addAction(getAction(title: "UIWebView", controllerID: "uiWebID"))
```

Comment:
You already have `ActionType. titleAndControllerID` - why not use them?
Also, I am not sure if you really need that `And`. I would rather suggest to split `title` an `controllerId` into two different properties.